core/src/test/java/kafka/admin/ClientTelemetryTest.java

```diff
@@ -124,6 +128,20 @@ public void testClientInstanceId(ClusterInstance clusterInstance) throws Interru
         }
     }
 
+    private static String[] toArray(List<String>... lists) {
+        return Stream.of(lists).flatMap(List::stream).toArray(String[]::new);
+    }
```

Comment:
nit: would be better to place the `private` method after `public` methods and before static inner class.

Reply:
Thanks. I will notice in future.